CHANGELOG.md

```diff
@@ -16,7 +16,8 @@
 
 ##### Enhancements
 
-* None.
+* Return `SWIFT_NAME` when generating Objective-C docs.
```

Comment:
Missing two trailing spaces, as documented in [CONTRIBUTING.md](https://github.com/jpsim/SourceKitten/blob/master/CONTRIBUTING.md#tracking-changes).

Reply:
Done.